pyproject.toml

```diff
@@ -141,9 +141,8 @@ moto = "1.3.7"
 pip-licenses = "^3.5.1"
 pipdeptree = "^2.1.0"
 # TODO: Investigate whether a major version upgrade is allowable for 'pytest', which is several versions behind.
-pytest = ">=3.10,<4"
+pytest = "^5.0.0"
 pytest-cov = ">=2.2.1"
```

Comment:
I would have just commented-out pytest-exact-fixtures for a while so we have some temporary memory that it was there in case a question arises later. can always dig in git, but i find that a hassle. YMMV.